R/merPlots.R

```diff
@@ -36,22 +40,25 @@ plotREsim <- function(data, scale = 1.96, var = "median_eff",
     hlineInt <- 1
   }
   if(labs == TRUE){
-    xlabs.tmp <- element_text(face = "bold")
-    xvar <- labs
+    xlabs.tmp <- element_text(face = "bold", angle=90, vjust=.5)
+    data <- dplyr::ungroup(dplyr::arrange_(dplyr::group_by(data, level, effect), var))
+    data[,"xvar"] <- factor(data$group, levels=unique(data$group))
   } else {
     xlabs.tmp <- element_blank()
-    xvar <- "id"
+    data <- dplyr::ungroup(dplyr::mutate(dplyr::arrange_(dplyr::group_by(data, level, effect), var), xvar=dplyr::row_number(effect)))
```

Comment:
Do we have to use dplyr here? I am OK with doing so, but if we do that we will need to add it as a package dependency and increase the number of packages we depend upon. That's OK with me if this is easier and then it will allow us to do other things in other parts of the code better. Just want to confirm.